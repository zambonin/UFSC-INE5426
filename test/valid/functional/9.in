lambda int ref ref x -> ref ref x + 1
