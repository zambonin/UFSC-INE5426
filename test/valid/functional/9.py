exec(open('../../../src/scope_manager.py', 'r').read())
def Î»(x):
  return (x + 1)

