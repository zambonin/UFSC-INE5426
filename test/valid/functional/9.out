int fun: lambda (params: int ref ref x)
  ret + [ref] [ref] x 1
